docs/02_cleos/03_command-reference/create/account.md

```diff
@@ -1,7 +1,28 @@
+## Command 
+
+```ssh
+cleos create account [OPTIONS] creator name OwnerKey [ActiveKey
+```
+
+**Where**:
+
+* [`OPTIONS`] = See **Options** in **Command Usage** section below
+* `creator` = The name of the existing account creating the new account
+* `name` = The name of the new account
+* `OwnerKey` = The owner public key or permission level for the new account
+* [`ActiveKey`]= See **Positionals** in **Command Usage** section below
+
+[[info | Note]]
+| The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Create a new account on the blockchain (assumes system contract does not restrict RAM usage)
+Use this command to create a new blockchain account on EOSIO. 
 
-## Usage
+[[info | Note]]
+| This command assumes system contract does not restrict RAM usage. 
```

Comment:
I think this needs an explanation -> or a link to something explaining ram and accounts, and an accounts requirement for ram and the cost etc